authz/providers/azure/azure.go

```diff
@@ -49,7 +50,8 @@ func init() {
 }
 
 type Authorizer struct {
-	rbacClient *rbac.AccessInfo
+	rbacClient           *rbac.AccessInfo
+	HttpClientRetryCount int
```

Comment:
should this property be exported (public)?

Reply:
fixed